specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/capacityReservation.json

```diff
@@ -730,6 +730,27 @@
           ],
           "readOnly": true,
           "description": "List of instance view of the capacity reservations under the capacity reservation group."
+        },
+        "sharedSubscriptionIds": {
+          "type": "array",
+          "items": {
+            "$ref": "../../../common-types/v1/common.json#/definitions/SubResourceReadOnly"
+          },
+          "x-ms-identifiers": [],
+          "readOnly": true,
+          "description": "List of the subscriptions that the capacity reservation group is shared with. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details."
+        }
+      }
+    },
+    "ResourceSharingProfile": {
+      "type": "object",
+      "properties": {
+        "subscriptionIds": {
+          "type": "array",
+          "description": "Specifies an array of Subscription resource Ids that capacity reservation group is shared with. The resource Id should be provided in form: /subscriptions/{subscriptionId}. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details.",
```

Comment:
Would calling them 'subscriptionResourceIds' be helpful to developers for understanding they need to pass the full resource id instead of just the guid?

Reply:
(or really, anything else except just subscriptionIds!)